components/micro-gateway-cli/src/main/resources/templates/jwtRevocation.mustache

```diff
@@ -0,0 +1,11 @@
+map<string> receivedRevokedTokenMap = gateway:getRevokedTokenMap();
```

Comment:
Formatted correctly?

Reply:
<img width="798" alt="Screen Shot 2019-04-18 at 10 27 25 PM" src="https://user-images.githubusercontent.com/23296184/56377987-57ef6a00-6229-11e9-93ce-94e8d1884064.png">

yes, after code generation main.bal file will look like this